### PR TITLE
Improve search engine optimization

### PR DESCRIPTION
**Pull request by Stinggyray**
there's not a lot of changes here lol

added robots.txt and fixed some other things

### CODE REVIEW

File: src/Layout.tsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ const Layout: React.FC<LayoutProps> = (props) => {
           content={`${siteUrl}${image ?? defaultImage}`}
         />
         <meta
-          property="description"
+          name="description"
```
Comment (iansjk) [inline src/Layout.tsx:108]: We still have other `<meta property=...` tags above this one; per MDN, `property` isn't a supported attribute on `<meta>` tags: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta

**Review by iansjk — CHANGES_REQUESTED** [review]

File: src/Layout.tsx (hunk 1 above)
Comment (Stinggyray) [reply src/Layout.tsx:108]: it's what Open Graph Protocol uses so go yell at them

if you remove them embeds will break
https://ogp.me/

image

Comment (iansjk) [reply src/Layout.tsx:108]: As the kids say, that is *wack*

**Review by iansjk — APPROVED** [review]
(Note: the `ArknightsGameData` submodule is updated in this PR; it should have been updated in the previous PR that fixed nested tags in `description-parser.ts`, but we've done it here instead.)

File: src/Layout.tsx (hunk 1 above)
Comment (Stinggyray) [reply src/Layout.tsx:108]: Straight wack, yo no cap

**Pull request merged**